furniro-front-end/src/app/header/components/HeaderButtons.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Account from "../images/account.png";
+import Search from "../images/search.png";
+import Favorites from "../images/favorites.png";
+import Cart from "../images/cart.png";
+import Image, { StaticImageData } from "next/image";
+
+interface imageProps {
+  src: string | StaticImageData;
+  alt: string;
+}
+
+const HeaderButtons = () => {
+  const buttons = [
+    {
+      name: "Home",
+      alt: "Home",
+    },
+    {
+      name: "Shop",
+      alt: "Shop",
+    },
+    {
+      name: "About",
+      alt: "About",
+    },
+    {
+      name: "Contact",
+      alt: "Contact",
+    },
+  ];
+  const images: imageProps[] = [
+    {
+      src: Account,
+      alt: "Account",
+    },
+    {
+      src: Search,
+      alt: "Search",
+    },
+    {
+      src: Favorites,
+      alt: "Favorites",
+    },
+    {
+      src: Cart,
+      alt: "Cart",
+    },
+  ];
```

Comment:
Create a data folder inside the Header module, create a data file and export it from there, you don't need to use this kind of code directly in your components